src/views/App/index.tsx

```diff
@@ -37,6 +38,7 @@ class App extends React.Component<MainProps, MainState> {
                 path="/system/:profileId"
                 component={SystemProfile}
               />
+              <SecureRoute path="/new" component={SystemIntake} />
```

Comment:
This is probably a larger product discussion as a whole about how we define URL structure. `/new` probably too generic though.

Reply:
I think I made a comment in one of your PRs to the same effect...